modules/stochastic_tools/src/vectorpostprocessors/StochasticResults.C

```diff
@@ -34,8 +34,6 @@ StochasticResults::StochasticResults(const InputParameters & parameters)
 void
 StochasticResults::initialize()
 {
-  mooseAssert(_sampler, "The _sampler pointer must be initialized via the init() method.");
```

Comment:
Why are you removing this? I added this in case someone wants to create a different transfer that populates a StochasticResults object. This will help make sure that init() is called on the result object.

Reply:
It was deleted accidentally. I just added it back. thanks!